the-blue-alliance-ios/Frameworks/TBAKit/Testing/MockTBAKit.swift

```diff
@@ -1,12 +1,14 @@
+import TBAKit
 import XCTest
-@testable import The_Blue_Alliance
 
-class MockTBAKit: TBAKit {
+public class MockTBAKit: TBAKit {
 
-    let session: MockURLSession
+    public let session: MockURLSession
+    private let bundle: Bundle
 
-    init(userDefaults: UserDefaults) {
+    public init(userDefaults: UserDefaults, bundle: Bundle) {
```

Comment:
This is problematic... at least, it's failing our tests in the TBA unit tests. I think we might consider having some fake responses we use for these inside of TBA and move on. Feels easiest, and also makes sense. It allows us to control responses between the TBAKit tests and the TBA tests